test/handler/async2.test.mo

```diff
@@ -51,19 +58,19 @@ let fut1 = async { await* handler.fetchFee() };
 await ledger.transfer_.clear();
 
 // stage a second response
-let release2 = ledger.fee_.stage(?10);
+let release2 = ledger.fee_.stage(null);
```

Comment:
I think we can remove staging the second response entirely. That response will never be picked up anyway. Because the second call to handler.fetchFee() will return immediately with `null` without making a downstream call.